web/src/views/settings/ClassificationSettingsView.tsx

```diff
@@ -170,31 +170,22 @@ export default function ClassificationSettingsView({
         );
       })
       .finally(() => {
+        addMessage(
+          "search_settings",
+          `Unsaved Classification settings changes`,
```

Comment:
This should probably just be `"Restart required (classification settings changed)"`